backend/README.md

```diff
@@ -20,3 +33,7 @@ REDIS_PORT=6379
 Have a look into the main `README.md` file for more information on how to set up
 the backend for development. Use the scripts within the
 scripts/backend-development folder to run the BE API locally.
+
+## Alembic
+
+To the database run `alembic revision -m "..."` ("..." is what you did) in the `/backend` directory. Then edit the newly created file. See [here](https://alembic.sqlalchemy.org/en/latest/tutorial.html) for more information. 
```

Comment:
This should be `alembic revision --autogenerate -m "..."` ... and I think at the beginning of the sentence it should be "To create an Alembic database migration script after sql-models were modified run `alembic revision --autogenerate -m "..."`"